src/persist/src/location.rs

```diff
@@ -77,20 +80,52 @@ pub fn check_meta_version_maybe_delete_data<B: Blob>(b: &mut B) -> Result<(), Er
 /// Read-only requests are assigned the SeqNo of a write, indicating that all
 /// mutating requests up to and including that one are reflected in the read
 /// state.
-#[derive(Clone, Copy, Debug, Default, PartialOrd, Ord, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(
+    Clone, Copy, Debug, Default, PartialOrd, Ord, PartialEq, Eq, Hash, Serialize, Deserialize,
+)]
 pub struct SeqNo(pub u64);
 
+impl std::fmt::Display for SeqNo {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "v{}", self.0)
+    }
+}
+
 impl timely::PartialOrder for SeqNo {
     fn less_equal(&self, other: &Self) -> bool {
         self <= other
     }
 }
 
+impl Timestamp for SeqNo {
```

Comment:
Do we need the `Timestamp` impl (which requires `PathSummary` and `Hash`) for follow-up changes? It seems we only need `minimum()` right now, which could just be in an `impl SeqNo {}`.

Reply:
good call, we don't need it to actually be a Timestamp yet and I'm not sure we actually ever do. i'll update this

Reply:
done